graphics/vtk/Portfile

```diff
@@ -14,8 +14,8 @@ compiler.cxx_standard 2011
 compiler.blacklist-append {clang < 900}
 
 name                vtk
-version             8.2.0
-revision            11
+version             9.1.0
+revision            1
```

Comment:
You'll want to change `revision` to zero.

Reply:
Oops! Have changed to 0, rebased to current master and squashed.